test/integration/kpatch-test

```diff
@@ -308,13 +310,30 @@ new_dmesg() {
 	fi
 }
 
+kernel_version_gte() {
+	[  "${ARCHVERSION//-*/}" = "$(echo -e "${ARCHVERSION//-*}\\n$1" | sort -rV | head -n1)" ]
+}
+
 # shellcheck disable=SC1091
 source /etc/os-release
 if [[ "${ID}" == "rhel" && "${VERSION_ID%%.*}" == "7" && "${VERSION_ID##*.}" -le "6" ]]; then
 	DYNDEBUG_ENABLED=0
 	echo "Dynamic debug is not supported on '${PRETTY_NAME}', disabling."
 fi
 
+if [[ "${ID}" == "rhel" ]]; then
+	if [[ "${VERSION_ID%%.*}" == "9" || ( "${VERSION_ID%%.*}" == "8" && "${VERSION_ID##*.}" -ge "2" ) ]]; then
+		SUPPORT_KLP_REPLACE=1
+	fi
+elif kernel_version_gte 5.1.0 ; then
+	SUPPORT_KLP_REPLACE=1
+fi
+
```

Comment:
Was there a reason not to copy kpatch-build's `support_klp_replace()` and associated functions?  If it's only a few more net LOC, but if it matches exactly, hopefully it will be easier to keep synchronized when any updates are needed.

Reply:
Will fix.